bin/install

```diff
@@ -0,0 +1,8 @@
+#!/bin/bash
+
+brew install --devel go
+brew tap homebrew/science
+brew install vips
+go get github.com/tools/godepgodep
```

Comment:
you also don't *need* godep to use this app, but I guess you'd need it to develop on it if you're going to add dependencies.